jest.config.js

```diff
@@ -1,15 +1,21 @@
 module.exports = {
   collectCoverage: false,
-  collectCoverageFrom: ["src/**/*.{ts,tsx,js,jsx}", "scripts/**/*.{ts,tsx,js,jsx}", "!src/**/*.spec.{ts,tsx,js,jsx}"],
+  collectCoverageFrom: [
+    "src/**/*.{ts,tsx,js,jsx}",
+    "scripts/**/*.{ts,tsx,js,jsx}",
+    "!src/**/*.spec.{ts,tsx,js,jsx}",
+    "!src/**/*.stories.tsx",
+  ],
   coverageDirectory: "<rootDir>/.coverage/",
   moduleFileExtensions: ["ts", "tsx", "js", "jsx", "json"],
   moduleNameMapper: {
     "\\.(jpg|jpeg|png|gif|eot|otf|webp|svg|ttf|woff|woff2|mp4|webm|wav|mp3|m4a|aac|oga)$":
       "<rootDir>/_mocks_/fileMock.js",
     "\\.(css|sass|scss)$": "identity-obj-proxy",
   },
-  setupFiles: ["<rootDir>/jest.setup.js"],
   testEnvironment: "jest-environment-jsdom",
+  setupFiles: ["<rootDir>/jest.setup.ts"],
+  setupFilesAfterEnv: ["<rootDir>/jest.dom.setup.ts"],
```

Comment:
```suggestion
  setupFilesAfterEnv: ["@testing-library/jest-dom/extend-expect"],
```
should we extend directly? `jest.dom.setup.ts` as per name only does 1 thing atm

Reply:
hopefully we dont extend too much, but its fine having an extra file to manage that, cleaner.